web/src/components/Header/HeaderLeftPanel.tsx

```diff
@@ -112,6 +116,44 @@ const HeaderLeftPanel: React.FC<HeaderLeftPanelProps> = ({
   useOnClickOutside(ref, () => setOpenEntryPointPicker(false))
   const [openEntryPointPicker, setOpenEntryPointPicker] = useState(false)
 
+  const [numOpsToFetch, setNumOpsToFetch] = useState(0)
+
+  // display syncing indicator when numOpsToFetch > 0
+  useEffect(() => {
+    const fetchOpData = async () => {
+      const adminWs = await getAdminWs()
+      const dnas = await adminWs.listDnas()
+      const appWs = await getAppWs()
+      const networkInfo = await appWs.networkInfo({
+        agent_pub_key: agentAddress as any,
+        dnas: dnas as any,
+      })
+
+      if (projectId === null || projectId === undefined) {
+        return
+      }
+
+      let i: number
+      for (i = 0; i < dnas.length; i++) {
+        const dna = dnas[i].toString()
+        const activeProjectDna = projectId.substring(0, projectId.indexOf('['))
+        if (dna === activeProjectDna) break
+      }
+
+      if (i >= dnas.length) {
+        return
+      }
```

Comment:
how could this occur?

Reply:
i dont think it ever *should*, but it could in theory occur if the active project DNA was not in the list of DNAs at all

Reply:
That can’t occur so we could remove it